chapters/en/slides/module4/module4_02.md

```diff
@@ -0,0 +1,122 @@
+---
+type: slides
+title: 'module4_02'
+---
+
+# Restoring an older version of a file
+
+---
+# Time travelling 
+
+
+We are going to learn two ways you to go back in time to a previous commits:
+
+
+1. **reset hard** 
+2. **revert** 
+
+Notes: Oh no! We realize by viewing the history that we made a mistake! We didn't mean to delete our list of dependencies. Worry not! We can now take advantage that we have been tracking this file under version control to retrieve an older version of the file to replace the current version.
+
+
+
+---
+# Refer to a commit
+
+Commit ID or SHA-1 hash
+
+```bash
+commit b22d6421363c7b82e2b88b0b5f1f25edca703303
+commit b22d
+```
+Notes: The commit ID or SHA-1 identifies the commit. the commit ID will be necessary to can identify the change we want to modify.
+
+---
+
+
+
+# 2. Reset your changes: delete your changes AND the history of your project (JupyterLab)
```

Comment:
I think it's better to cut "_delete your changes AND the history of your project (JupyterLab)_" from the slide title and paste it into the slide itself.

Reply:
I will keep this comment for further discussion